.lintstagedrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  // standard's configuration will let it ignore built files
+  // in `dist` or `package` that we want left alone, as well as the polyfills
+  '*.{js,mjs}': ['standard --fix'],
```

Comment:
Thoughts on keeping things in one place, and only appending `-- --fix` to the existing npm scripts?

On other projects I've had "default" scripts, plus others ready to accept arguments:

```json
"lint:js": "npm run lint:js:cli -- \"**/*.{cjs,js,mjs}\"",
"lint:js:cli": "standard",
```

```suggestion
  '*.{cjs,js,mjs}': ['npm run lint:js -- --fix'],
```

Reply:
I'm guessing you meant using `npm run lint:js:cli` in the `lintstagedrc`. No strong opinion on that one for me, just a bit warry of adding levels of indirection before a command actually runs. It does make updates more centralised, though, which is nice 😄 

Cheers for picking up the lack of `cjs` extension in the `lintstagedrc`, that'll future-proof that file (though unless we move to having`type="module"` in package.json, `.cjs` files shouldn't appear in the codebase, should they?) 😄

Reply:
I did indeed 🤦‍♂️ 

They shouldn't need to appear no, but if they do they won't be linted on commit 😉 

Mainly for completeness that "All JavaScript will be linted" (it's in our ESLint config too)